controller/restore_controller.go

```diff
@@ -43,6 +43,12 @@ func (r *RestoreReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 	}
 
 	mariaDb, err := r.RefResolver.MariaDB(ctx, &restore.Spec.MariaDBRef, restore.Namespace)
+
+	if mariaDb.Spec.BootstrapFrom != nil && *mariaDb.Spec.BootstrapFrom.Type == "mariabackup" {
+		// skip restore reconciler for the mariadbbackup restore
```

Comment:
What about about deleting the `/var/lib/mysql/restore_success` from the Job and restoring the backup? Afterwards the `Pod` should be restarted.

This could be helpful is situations where the replicas go down and need a physical backup to be restored. For example:
- https://github.com/mariadb-operator/mariadb-operator/issues/141